app/models/orchestration_template.rb

```diff
@@ -153,7 +153,7 @@ def md5=(_md5)
   end
 
   def self.calc_md5(text)
-    Digest::MD5.hexdigest(text) if text
+    Rails.application.config.digest_class.hexdigest(text) if text
```

Comment:
agree about better OOTB behavior, however the driver behind this is FIPS and I can see a point in time when FIPS precludes SHA2 as well.  Perhaps I am being overly cautious